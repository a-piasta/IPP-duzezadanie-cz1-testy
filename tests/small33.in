getRouteDescription;236
newRoute;109;dgdwfnejwbqyvyfd;rbkocrtsuiqaoxxbtingxiwsllvz
addRoad;rbkocrtsuiqaoxxbtingxiwsllvz;bpwgffxlwijlpvqvbmaoqj;17;-15
newRoute;619;rbkocrtsuiqaoxxbtingxiwsllvz;rbkocrtsuiqaoxxbtingxiwsllvz
removeRoad;rbkocrtsuiqaoxxbtingxiwsllvz;bpwgffxlwijlpvqvbmaoqj
removeRoad;rbkocrtsuiqaoxxbtingxiwsllvz;bpwgffxlwijlpvqvbmaoqj
getRouteDescription;109
getRouteDescription;619
repairRoad;dgdwfnejwbqyvyfd;qwtooarqyhicjvhcugh;3
getRouteDescription;619
addRoad;bpwgffxlwijlpvqvbmaoqj;osswpcukoffphldevap;0;2
removeRoad;bpwgffxlwijlpvqvbmaoqj;rbkocrtsuiqaoxxbtingxiwsllvz
addRoad;rbkocrtsuiqaoxxbtingxiwsllvz;pwkgdqhogrlwne;9;-14
newRoute;123;npemxrxhyuxuhgnmrcbsuvkyfg;dgdwfnejwbqyvyfd
repairRoad;pwkgdqhogrlwne;glvkxjmodeilwlbti;3
removeRoad;bpwgffxlwijlpvqvbmaoqj;rbkocrtsuiqaoxxbtingxiwsllvz
extendRoute;109;npemxrxhyuxuhgnmrcbsuvkyfg
removeRoad;rbkocrtsuiqaoxxbtingxiwsllvz;pwkgdqhogrlwne
removeRoad;pwkgdqhogrlwne;rbkocrtsuiqaoxxbtingxiwsllvz
newRoute;36;qwtooarqyhicjvhcugh;npemxrxhyuxuhgnmrcbsuvkyfg
addRoad;bpwgffxlwijlpvqvbmaoqj;qwtooarqyhicjvhcugh;21;0
addRoad;pspzglkjgvoun;bpwgffxlwijlpvqvbmaoqj;26;14
extendRoute;50;pspzglkjgvoun
addRoad;pwkgdqhogrlwne;pspzglkjgvoun;29;-3
addRoad;pspzglkjgvoun;glvkxjmodeilwlbti;2;-13
addRoad;npemxrxhyuxuhgnmrcbsuvkyfg;osswpcukoffphldevap;3;-15
newRoute;971;dgdwfnejwbqyvyfd;bpwgffxlwijlpvqvbmaoqj
repairRoad;bpwgffxlwijlpvqvbmaoqj;qwtooarqyhicjvhcugh;-9
repairRoad;npemxrxhyuxuhgnmrcbsuvkyfg;glvkxjmodeilwlbti;0
newRoute;646;sdlnpzsuqbhtffwv;rbkocrtsuiqaoxxbtingxiwsllvz
repairRoad;glvkxjmodeilwlbti;bpwgffxlwijlpvqvbmaoqj;3
newRoute;315;pwkgdqhogrlwne;npemxrxhyuxuhgnmrcbsuvkyfg
removeRoad;qwtooarqyhicjvhcugh;bpwgffxlwijlpvqvbmaoqj
extendRoute;619;dgdwfnejwbqyvyfd
repairRoad;osswpcukoffphldevap;osswpcukoffphldevap;-1
removeRoad;bpwgffxlwijlpvqvbmaoqj;pspzglkjgvoun
addRoad;pwkgdqhogrlwne;bpwgffxlwijlpvqvbmaoqj;3;-13
repairRoad;rbkocrtsuiqaoxxbtingxiwsllvz;bpwgffxlwijlpvqvbmaoqj;12
getRouteDescription;123
getRouteDescription;109
extendRoute;971;osswpcukoffphldevap
extendRoute;50;osswpcukoffphldevap
newRoute;348;sdlnpzsuqbhtffwv;bpwgffxlwijlpvqvbmaoqj
newRoute;493;dgdwfnejwbqyvyfd;rbkocrtsuiqaoxxbtingxiwsllvz
newRoute;12;dgdwfnejwbqyvyfd;npemxrxhyuxuhgnmrcbsuvkyfg
removeRoad;bpwgffxlwijlpvqvbmaoqj;osswpcukoffphldevap
addRoad;qwtooarqyhicjvhcugh;rbkocrtsuiqaoxxbtingxiwsllvz;29;2
newRoute;987;qwtooarqyhicjvhcugh;pwkgdqhogrlwne
extendRoute;348;pspzglkjgvoun
repairRoad;rbkocrtsuiqaoxxbtingxiwsllvz;bpwgffxlwijlpvqvbmaoqj;-8
newRoute;696;osswpcukoffphldevap;pspzglkjgvoun
removeRoad;bpwgffxlwijlpvqvbmaoqj;pspzglkjgvoun
removeRoad;rbkocrtsuiqaoxxbtingxiwsllvz;bpwgffxlwijlpvqvbmaoqj
getRouteDescription;348
newRoute;166;sdlnpzsuqbhtffwv;qwtooarqyhicjvhcugh
repairRoad;pwkgdqhogrlwne;dgdwfnejwbqyvyfd;5
addRoad;pwkgdqhogrlwne;bpwgffxlwijlpvqvbmaoqj;2;-2
addRoad;pspzglkjgvoun;glvkxjmodeilwlbti;19;6
removeRoad;osswpcukoffphldevap;npemxrxhyuxuhgnmrcbsuvkyfg
repairRoad;osswpcukoffphldevap;pwkgdqhogrlwne;10
