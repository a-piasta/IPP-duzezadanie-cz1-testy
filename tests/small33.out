1: 
2: NIE
3: TAK
4: NIE
5: TAK
6: NIE
7: 
8: 
9: NIE
10: 
11: NIE
12: NIE
13: TAK
14: NIE
15: NIE
16: NIE
17: NIE
18: TAK
19: NIE
20: NIE
21: NIE
22: TAK
23: NIE
24: TAK
25: TAK
26: TAK
27: NIE
28: NIE
29: NIE
30: NIE
31: NIE
32: NIE
33: NIE
34: NIE
35: NIE
36: TAK
37: TAK
38: NIE
39: 
40: 
41: NIE
42: NIE
43: NIE
44: NIE
45: NIE
46: NIE
47: TAK
48: NIE
49: NIE
50: NIE
51: NIE
52: NIE
53: NIE
54: 
55: NIE
56: NIE
57: NIE
58: NIE
59: TAK
60: NIE
